gulpfile.js

```diff
@@ -1,17 +1,30 @@
 var gulp = require('gulp');
 var sass = require('gulp-sass');
 var minifyHtml = require("gulp-minify-html");
+var concat = require("gulp-concat");
 var browserSync = require('browser-sync').create();
 
+gulp.task('js', function () {
+    return gulp.src([
+        './bower_components/jquery/dist/jquery.min.js',
+        './bower_components/materialize/dist/js/materialize.min.js',
+        './src/**/*.js'
+    ])
+        .pipe(concat('scripts.js'))
+        .pipe(gulp.dest('./dist'))
+        .pipe(browserSync.stream());
+});
```

Comment:
Das hier ist eigentlich das gleiche wie bei `sass` oder `html`, nur dass statt dem Unwandeln in CSS bzw. Minify hier ein "concat" der Dateien erfolgt. Danach wird wieder Browser-Sync benachrichtigt, dass sich etws geändert hat.